controllers/llamaCPP.h

```diff
@@ -2277,5 +2278,8 @@ class llamaCPP : public drogon::HttpController<llamaCPP>, public ChatProvider {
   void warmupModel();
   void backgroundTask();
   void stopBackgroundTask();
+  void model_loaded_response(
```

Comment:
I am refactoring the coding conventions of these controllers. Please refer to issue #444, where it is mentioned that class methods should follow PascalCase.